include/libtorrent/alert_types.hpp

```diff
@@ -694,7 +694,7 @@ TORRENT_VERSION_NAMESPACE_3
 		// 2. Started
 		// 3. Stopped
 		// 4. Paused
-		event_t const event;
+		aux::event_t const event;
```

Comment:
hi @arvidn, this one here, I don't know how do you want to proceed. new header for that enum only? remove this? `event_t` is "documented" as internal

Reply:
is `event_t` a public type?

Reply:
oh, I see. It's public right here.. I think this type should be public, and properly documented. I think a new header for just this enum, or some other appropriate header would be fine. There's some precedence for putting types like these in its own header, such as `socket_type_t` and `operations_t`